lib/game.rb

```diff
@@ -1,24 +1,35 @@
+require_relative "grid_view"
+require_relative "player_turn"
+require_relative "text_view"
+
 class Game
-  CLI_GRID = <<~GRID.freeze
-       1  2  3
-       __ __ __
-    A |  |  |  |
-      |__|__|__|
-    B |  |  |  |
-      |__|__|__|
-    C |  |  |  |
-      |__|__|__|
-  GRID
-
-  def initialize(output:)
+  def initialize(input:, output:)
+    @input = input
     @output = output
   end
 
   def play
-    output.puts(CLI_GRID)
+    grid_view.render
+    get_player_input
   end
 
   private
 
-  attr_reader :output
+  attr_reader :input, :output
+
+  def get_player_input
+    player_input.get_coordinates
+  end
+
+  def player_input
+    PlayerTurn.new(input: input, view: text_view)
+  end
+
```

Comment:
Nice! Seeing these view objects is cool. Also, when I see a top level coordinator class coalescing into these object instantiations and then DI, it feels good.